backend/src/youtube-feed/youtube-feed.service.ts

```diff
@@ -30,20 +38,32 @@ export class YouTubeFeedService {
     }
 
     onModuleInit() {
-        this.fetchAndSaveYouTubeFeed();
+        setInterval(() => {
+            this.fetchAndSaveYouTubeFeed();
+        }, 1000);
     }
 
     async fetchAndSaveYouTubeFeed() {
         this.logger.log('Fetching Youtube feeds Started');
         try {
-            const feed = await axios.get(youtube_fetch_url);
-            const videoIds = feed.data.items.map(item => item.id.videoId);
-            const videoDetails = await axios.get(youtube_video_details_url + videoIds.filter(id => id));
-            const youTubeFeeds = feed.data.items.map(item => {
+            const channelDetails = await axios.get(channelDetailsUrl);
+
+            const channelTitle = channelDetails.data.items[0].snippet.title;
```

Comment:
If there is no item this code will break, because of how JS work. You might wanna use a getter